query/src/main/scala/filodb/query/exec/AggrOverRangeVectors.scala

```diff
@@ -934,72 +935,72 @@ class QuantileRowAggregator(q: Double) extends RowAggregator {
   }
 }
 
+/**
+  * Map: Every sample is mapped to map with value as key and 1 as value
```

Comment:
"value as key and 1 as value" sounds funny :D 
I get you, but consider rephrasing for others.